sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/io/sermant/flowcontrol/DispatcherServletInterceptor.java

```diff
@@ -71,12 +70,12 @@ private Optional<HttpRequestEntity> convertToHttpEntity(HttpServletRequest reque
      * @param request request information
      * @return headers
      */
-    private Map<String, String> getHeaders(HttpServletRequest request) {
-        final Enumeration<String> headerNames = request.getHeaderNames();
+    private Map<String, String> getHeaders(Object request) {
+        final Enumeration<String> headerNames = getHeaderNames(request);
```

Comment:
use class name to distinguish low version and high version

Reply:
Use the loadclass method to determine the version.